cmd/root.go

```diff
@@ -0,0 +1,78 @@
+package cmd
+
+import (
+	"log"
+	"os"
+	"time"
+
+	"github.com/spec-tacles/spectacles.go/broker"
+	"github.com/spec-tacles/spectacles.go/gateway"
+	"github.com/spec-tacles/spectacles.go/rest"
+	"github.com/spec-tacles/spectacles.go/types"
+	"github.com/spf13/cobra"
+)
+
+var amqpUrl, amqpGroup, token, logLevel string
+var shardCount int
+
+var brokerConnected bool
+
+var logger = log.New(os.Stdout, "[CMD] ", log.Ldate|log.Ltime|log.Lshortfile)
+var logLevels = map[string]int{
+	"suppress": gateway.LogLevelSuppress,
+	"info":     gateway.LogLevelInfo,
+	"warn":     gateway.LogLevelWarn,
+	"debug":    gateway.LogLevelDebug,
+	"error":    gateway.LogLevelError,
+}
+
+var RootCmd = &cobra.Command{
+	Use:   "spectacles",
+	Short: "Connects to the Discord websocket API using spectacles.go",
+	Run: func(cmd *cobra.Command, args []string) {
+		amqp := broker.NewAMQP(amqpGroup, "", func(string, []byte) {})
+		tryConnect(amqp)
+
+		manager := gateway.NewManager(&gateway.ManagerOptions{
+			ShardOptions: &gateway.ShardOptions{
+				Identify: &types.Identify{
+					Token: token,
+				},
+			},
+			OnPacket: func(shard int, d *types.ReceivePacket) {
+				if brokerConnected {
```

Comment:
This condition is unnecessary now that the broker is always connected before receiving packets.